docs/resources/vpc_gcp_peering.md

```diff
@@ -268,4 +277,18 @@ resource "cloudamqp_security_firewall" "firewall_settings" {
   ]
 }
 ```
+
 </details>
+
+## Changelog
+
+List of changes made to this resource for different versions.
+
+[v1.29.0](https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.29.0) configurable
+sleep and timeout for retries when requesting and reading peering.
+
+[v1.28.0](https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.18.0)
+Added `wait_on_peering_status` as optional, set to wait until peering is finished.
+
+[v1.16.0](https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.16.0)
+deprecated intance_id and use vpc_id instead
```

Comment:
I see we are discussing this at https://github.com/cloudamqp/terraform-provider-cloudamqp/pull/246#discussion_r1428028499 :)